File: PRAK504/PRAK504-2210817310013-Ryan Muhammad Irfan.py
def reverse(a, reversed=0):
    while (a != 0):
        reversed *= 10
        reversed += a % 10
        a //= 10
    return reversed


A, B = input("").split()
A = reverse(int(A))
B = reverse(int(B))
C = A + B
print(reverse(C))

File: PRAK504/PRAK504-2210817310013-RyanMuhammadIrfan.c
#include <stdio.h>
int reverse(int a, int reversed){
    while (a != 0){
        reversed *= 10;
        reversed +=  a%10;
        a /= 10;
    }
    return reversed;
}
int main(){
    int A, B,reversed=0;
    scanf("%d %d", &A, &B);
    A = reverse(A,reversed);
    B = reverse(B,reversed);
    int C = A + B;
    printf("%d", reverse(C,reversed));
}